### ISSUE TITLE
items in Order (many to many fields)

### ISSUE THREAD

**Issue by HenriPiot-dev** [justdjango/django-ecommerce#68]
I'm not sur that this is an issues, but i figured out something, when multiple user are creating cart (independently of course) every instance of OrderItem are display on the many to many fields of the Order class, in the admin. I wondered if you did that on purpose or if it is an issue, because it's not really convenient if you want to check the product ordered by someone ?

**Comment by samuel025** [comment]
If you want to check the order created by someone, check the Order model in the admin site

**Comment by aritra31** [comment]
Right, but it's showing all the orderitems in the field with the user specific order highlighted. I think it'll be convenient if only the higlighted items are shown. Can you suggest a way to do that?
